services/github/src/handlers/status.js

```diff
@@ -36,7 +36,7 @@ const qLock = new QueueLock({
  **/
 async function statusHandler(message) {
   const { taskGroupId, state, runs, taskId } = message.payload.status;
-  const { runId } = message.payload;
+  const { runId } = message.payload || 0;
```

Comment:
yeah, it is `undefined` since merging of taskDefined and status handlers .. I forgot to adjust this part. 
Before `taskDefined` was a separate handler and newly defined task never emitted `runId` as task wasn't even scheduled yet, so `status` and `runs` could have not existed at that time.
And in the new `statusHandler` it should be safe to assume that task will be scheduled sooner or later and if `runId` is missing, it means it will become `0`. 
So we would need to check if it's undefined and adjust it to be `0`